bndtools.core/src/org/bndtools/core/ui/wizards/shared/TemplateParamsWizardPage.java

```diff
@@ -156,10 +156,13 @@ private void updateValidation() {
 	}
 
 	private Control createFieldControl(Composite parent, final AttributeDefinition ad) {
+		int textType = SWT.BORDER;
 		switch (ad.getType()) {
+			case AttributeDefinition.PASSWORD :
+				textType |= SWT.PASSWORD;
```

Comment:
You should put in a comment to document the intended FALL-THROUGH here.

Reply:
Done